nvm.sh

```diff
@@ -640,7 +640,6 @@ nvm_list_aliases() {
       nvm_echo "${LTS_ALIAS}"
     fi
   done
-  return
```

Comment:
This is done intentionally to avoid any previous commands' exit codes from leaking through.